atomics/T1098.001/T1098.001.yaml

```
@@ -20,41 +20,53 @@ atomic_tests:
       description: Targeted Application
       type: String
       default: "SuperApplication"
-    CertificatePassword:
+    certificate_password:
       description: Password of the new certificate
       type: string
       default: Passw0rd
-    PathToCert:
+    path_to_cert:
       description: Path of the new certificate, locally stored 
       type: string
+      default: $env:TEMP
+  dependency_executor_name: powershell
+  dependencies:
+  - description: |
+      AzureAD module must be installed and imported.
+    prereq_command: |
+      if (Get-Module AzureAD) {exit 0} else {exit 1}
+    get_prereq_command: |
+      Install-Module -Name AzureAD -Force
+      Import-Module -Name AzureAD
   executor:
     command: |
-      $User = "#{username}"
-      $App_name = "#{application}"
       $PWord = ConvertTo-SecureString -String "#{password}" -AsPlainText -Force
-      $Credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $User, $Pword
+      $Credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList #{username}, $Pword
```

```suggestion
      $Credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList "#{username}", $Pword
```
In case it contains spaces